cartography/intel/rapid7/util.py

```diff
@@ -0,0 +1,303 @@
+"""
+cartography/intel/rapid7/util
+"""
+# pylint: disable=invalid-name,broad-except
```

Comment:
is this needed?

Reply:
same than above. pylint usage (docstring, silence errors)

Reply:
These concerns do seem valid, so please address them.